analysed-packages/libnice/version-0.1.18/libnice-0.1.18-OSS-disclosure.txt

```diff
@@ -0,0 +1,1506 @@
+================================================================================
+
+libnice-0.1.18
+
+--------------------------------------------------------------------------------
+
+================================================================================
+
+LICENSES 
+
+--------------------------------------------------------------------------------
+
+LicenseRef-Dual-license
```

Comment:
```suggestion
Dual-license
```

Reply:
Two things are important here:

1. There should be no LicenseRef- in the name of licenses in OSS-disclosure file. you can remove it via tool. Please add this line to your python config.yaml file `replacement_regex_txt : [LicenseRef-="", LicenseRef-fossology-=""]`
2. The license text should be something like this https://github.com/Open-Source-Compliance/package-analysis/blob/main/analysed-packages/acl/version-2.3.2/acl-2.3.2-OSS-disclosure.txt#L1508